### PR TITLE
konflux: fix hermetic build argument

### PR DESCRIPTION
**Pull request by syedriko**
## Description

Turn on hermetic builds in Konflux

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -111,6 +111,7 @@ dependencies = [
     "jinja2==3.1.4",
     "scikit-learn==1.5.1",
     "tqdm==4.66.5",
+    "starlett==0.37.2",
```
Comment (tisnik) [inline pyproject.toml:114]: was it added w/o updating lockfile?

**Review by tisnik — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (tisnik) [reply pyproject.toml:114]: btw where it package came from?

Comment (syedriko) [reply pyproject.toml:114]: We somehow ended up with 
```
starlette==0.38.6 \
    --hash=sha256:4517a1409e2e73ee4951214ba012052b9e16f60e90d73cfb06192c19203bbb05 \
    --hash=sha256:863a1588f5574e70a821dadefb41e4881ea451a47a3cd1b4df359d4ffefe5ead
```
in requirements.txt, which caused
```
    |   File "/app-root/ols/app/main.py", line 127, in log_requests_responses
    |     response_body = response.body
    |                     ^^^^^^^^^^^^^
    | AttributeError: '_StreamingResponse' object has no attribute 'body'
```
at run time.

Comment (syedriko) [reply pyproject.toml:114]: >was it added w/o updating lockfile?

Yes, I missed that part. I suggest going ahead with this PR and fixing the lock file later, as pdm is not used in the build.

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Review by tisnik — CHANGES_REQUESTED** [review]

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**